src/panel-ladder.tsx

```diff
@@ -48,13 +48,12 @@ class LadderRoom extends PSRoom {
 	};
 	requestLadderData = (searchValue?: string) => {
 		this.setLadderData(undefined); // "Loading..."
-		$.get('/ladder.php', { // TO REVIEW: I imagine this may need to be changed to use PSLoginServer, but that does not have support for prepping GETs with query parameters
+		Net('/ladder.php').get({query: {
 			format: this.selectedFormat,
```

Comment:
tbh maybe we should support `undefined` to mean "leave this off", considering how hard it is to have a property that's only conditionally there.

Reply:
I'm going to have an entirely separate room for the ladder table, so it's also entirely possible that the user joins a room corresponding to a format that does not even exist. I'll have to run through a few scenarios.